Nightscouter/SiteDetailViewController.swift

```diff
@@ -141,35 +141,30 @@ extension SiteDetailViewController {
     
     func updateSite(_ notification: Notification?) {
         print(">>> Entering \(#function) <<<")
-        self.updateData(forceUpdate: true)
+        self.updateData()
     }
     
-    func updateData(forceUpdate force: Bool = false) {
+    func updateData() {
         guard let site = self.site else { return }
+        UIApplication.shared.isNetworkActivityIndicatorVisible = true
         
-        if (site.updateNow || site.sgvs.isEmpty || force == true) {
```

Comment:
Right, so`force` was always equal to true. So this line read as follows:

`if (anything OR something else OR true) then do the thing`

Again, since that last boolean was always true, this entire expression was 100% of the time resolving to `true`, so the if clause was irrelevant.